### PR TITLE
vscode-insider: Update to version 1.87.0

### LINKED ISSUE #130: vscode-insiders: fix checkver
Error from the excavator
```text
vscode-insiders: 1.87.0-insider (scoop version is 1.84.0-insider) autoupdate available
Autoupdating vscode-insiders
Searching hash for VSCode-win32-ia32-1.87.0-insider.zip in https://code.visualstudio.com/sha?build=insider
Could not find hash in https://code.visualstudio.com/sha?build=insider
Downloading VSCode-win32-ia32-1.87.0-insider.zip to compute hashes!
The remote server returned an error: (400) Bad Request.
URL https://vscode.download.prss.microsoft.com/dbazure/download/insider/7c15c66607256592202cc051fb3e117baf54af63/VSCode-win32-ia32-1.87.0-insider.zip is not valid
ERROR Could not update vscode-insiders, hash for VSCode-win32-ia32-1.87.0-insider.zip failed!
```
Investigation is required

### PR DESCRIPTION
**Pull request by alexchapar**
## What type of Apps related PR is this? (check all applicable)
- [ ] ✨ Add Manifest
- [x] ♻️ Refactor
- [x] 🐛 Bug Fix
- [ ] 👷 Optimization
- [x] 🚩 Other
## Description
Fix the checkver regex
Update to the version 1.87.0
Enhance the manifest to include persist data
## Related Tickets & Documents
Fixes #130
## [optional] What gif/emoji best describes this PR or how it makes you feel?
🚑️♻️✨🎨
### Add/Modify a description in the manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Check URLs?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Update to the latest version?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Update Hashes?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Format the json manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Notes in the manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Application Dependencies?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Application Suggestions?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Pre-installation steps to perform?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Post-installation steps to perform?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### File persistence to perform?
- [x] 👍 yes ✨
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### vscode-insiders

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Review by Bibz87 — APPROVED** [review]

**Pull request merged**
